extra-tests/classes/RollupEvaluatorTests.cls

```diff
@@ -927,11 +887,14 @@ private class RollupEvaluatorTests {
 
   @IsTest
   static void shouldCorrectlyReformatNotLikeQuery() {
-    String whereClause = 'TaskSubType != \'Email\' OR Subject NOT LIKE \'Email\'';
+    String whereClause = 'TaskSubType != \'Email\' OR (NOT Subject LIKE \'Email\')';
 
     RollupEvaluator.WhereFieldEvaluator eval = new RollupEvaluator.WhereFieldEvaluator(whereClause, Task.SObjectType);
 
     System.assertEquals('TaskSubType != \'Email\' OR (NOT Subject LIKE \'Email\')', eval.getSafeQuery());
+    System.assertEquals(true, eval.matches(new Task(TaskSubType = 'Call')));
+    System.assertEquals(true, eval.matches(new Task(TaskSubType = 'Email', Subject = 'Hello')));
+    System.assertEquals(false, eval.matches(new Task(TaskSubType = 'Email', Subject = 'Email')));
```

Comment:
Yeah, those are all great points. I can definitely add in that extra assert. I've waffled back in forth on syntax but maybe best to just standardize to the new `Assert` class in a separate PR - in the meantime, I can update the asserts using the existing syntax to be consistent!

Reply:
Oops, realizing now - `new Task()` would technically be a _match_ at the moment since both of the clauses are negatives. I'll add that in there, though, I still think it's helpful to see.

Reply:
Oops, yeah, good call, it should be a match